scripts/run_all_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
+GOBIN="`pwd`/bin"
+KUBECONFIG="`pwd`/config"
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
```

Comment:
What's this step doing? Can we add a comment that would help us or someone who might look at this code few months later?

Reply:
Comments added